src/PathUtils.ts

```diff
@@ -516,4 +516,30 @@ export class PathUtils {
 
         return PathUtils.makeRelative(returnStr);
     }
+
+    /**
+     * Helper for handling remote react app routing in edit mode
+     * @param path Path to route to
+     * @param aemHost Origin information of the AEM instance in which to edit
+     * @param rootPath AEM path which forms the root path of the remote app
+     * @private
+     * @returns Updated url
+     */
+    public static toAEMPath(path: string, aemHost: string, rootPath: string): string {
```

Comment:
`toAEMPath` does not check if `path` already is prefixed, if you wrap path more than once with this function we will end up with a broken path